recipes/mdal-python/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+if [ "$(uname)" == "Linux" ]; then
+   export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
+fi
+
+set CMAKE_GENERATOR=Ninja
+${PYTHON} setup.py install -vv -- -DPython3_EXECUTABLE="${PYTHON}" --
```

Comment:
Have you tried using `${PYTHON} -m pip install -vv .` instead?

https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip#use-pip

Reply:
That won't work. Setup.py calls skbuild to compile the Cython libraries

Reply:
Assuming scikit-build is set up correctly in the package it should work and you get the benefit that `pip` is aware of the package: https://scikit-build.readthedocs.io/en/latest/usage.html#example-of-setup-py-cmakelists-txt-and-pyproject-toml

Reply:
You are, of course, correct although I had to make some small changes to the cmake files and do a version bump on the source repo to get it working. I have managed to remove the build.sh and bld.bat files. I think this commit will get through the tests.

Thanks for helping make the recipe considerably simpler and more compact.